packages/sdk/src/store/store.ts

```diff
@@ -139,7 +139,10 @@ export class Store {
         log('+enqueue load', tableName, id, loadPriority)
         this.checkTableName(tableName)
         check(this.transactionGroup !== undefined, 'transaction not started')
-        check(!this.isLoaded(tableName, id), `model already loaded table: ${tableName} id: ${id}`)
+        if (this.isLoaded(tableName, id)) {
+            log(`model already loaded table: ${tableName} id: ${id}`)
+            return
+        }
```

Comment:
can you revert this change? I know we need to do something for hot reloading, but i'd like to figure out what exactly is going on, because this breaks my model of how things should work.